resource_alks_ltk.go

```diff
@@ -11,6 +11,11 @@ func resourceAlksLtk() *schema.Resource {
 		Create: resourceAlksLtkCreate,
 		Read:   resourceAlksLtkRead,
 		Delete: resourceAlksLtkDelete,
+		Exists: resourceAlksLtkExists,
+		Importer: &schema.ResourceImporter{
```

Comment:
FWIW, I believe this is only used if you are changing the state format version.  We used something similar when we added machine identities as we had a new state field that we needed to set for existing roles.

Reply:
Ah I see now after reading some more on it. Do you think it is worth keeping this `exists` implementation, or maybe removing it to prevent confusion? I cannot imagine we'd change the state field any time soon...

Reply:
Let's leave it alone for the time being.  No harm there.